.tekton/rh-syft-pull-request.yaml

```diff
@@ -229,14 +238,18 @@ spec:
         value: $(tasks.prefetch-dependencies.results.SOURCE_ARTIFACT)
       - name: CACHI2_ARTIFACT
         value: $(tasks.prefetch-dependencies.results.CACHI2_ARTIFACT)
+      - name: PLATFORM
+        value: $(matrix.PLATFORM)
```

Comment:
Is `$(matrix.PLATFORM)` valid? I have a feeling the pipelineRun start failure is caused by this expression not getting expanded